hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/CompactionNode.java

```diff
@@ -41,18 +42,25 @@ public class CompactionNode {
 
   public CompactionNode(String file, long numKeys, long seqNum,
                         String startKey, String endKey, String columnFamily) {
+    this(file, numKeys, seqNum, startKey, endKey, columnFamily, false);
+  }
+
+  public CompactionNode(String file, long numKeys, long seqNum,
+                        String startKey, String endKey, String columnFamily,
+                        boolean pruned) {
     fileName = file;
     totalNumberOfKeys = numKeys;
     snapshotGeneration = seqNum;
     cumulativeKeysReverseTraversal = 0L;
     this.startKey = startKey;
     this.endKey = endKey;
     this.columnFamily = columnFamily;
+    this.pruned = pruned;
```

Comment:
We dont need this change from what I understand.

Reply:
Yes.